test/libsolidity/smtCheckerTests/crypto/crypto_functions_same_string_literal.sol

```diff
@@ -5,19 +5,25 @@ contract C {
 		assert(k1 == k2);
 	}
 
-        function c2() public pure {
+	function c2() public pure {
 		bytes32 s1 = sha256("10");
 		bytes32 s2 = sha256("10");
 		assert(s1 == s2);
 	}
 
-        function c3() public pure {
+	function c3() public pure {
 		bytes32 r1 = ripemd160("100");
 		bytes32 r2 = ripemd160("100");
 		assert(r1 == r2);
 	}
+
+	function c4() public pure {
+		address e1 = ecrecover("a", 1, "b", "c");
+		address e2 = ecrecover("a", 1, "b", "c");
```

Comment:
We could add different literal types for better coverage of corner cases:

```suggestion
		address e1 = ecrecover("a", 1, hex"b", unicode"c");
		address e2 = ecrecover("a", 1, hex"b", unicode"c");
```

Reply:
Changed